mainWindow.py

```diff
@@ -88,10 +91,14 @@ def on_runButton_clicked(self):
         #                         "before converting.")
         #     return
 
-        dataPath = "C:/Users/Clint/PycharmProjects/SIUE-Dixon-Fat-Segmentation-Algorithm/MRI_Data/MF0302-PRE/"
+        #dataPath = "/home/somecallmekenny/SIUE-Dixon-Fat-Segmentation-Algorithm/Data/Subject0001_Final/"
```

Comment:
somecallmekenny?

Should I call you kenny?

Reply:
lol it was a nickname in highschool and have used it as a screen name ever since. You can call me Sam